plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/listeners/CodeWhispererCodeScanEditorMouseMotionListener.kt

```diff
@@ -92,10 +92,11 @@ class CodeWhispererCodeScanEditorMouseMotionListener(private val project: Projec
             "-"
         }
 
-        val detectorLibraryLink = "<a href=\"https://docs.aws.amazon.com/codeguru/detector-library/${
-            issue.detectorId.split("@").first()
-        }\">${issue.detectorName}</a>"
-
+        val detectorLibraryLink = if (issue.recommendation.url != null) {
+            "<a href=\"${issue.recommendation.url}\">${issue.detectorName}</a>"
+        } else {
+            "-"
```

Comment:
is `-` or `N/A` better

Reply:
I used `-` since we are already doing that for cwes field above